### PR TITLE
[docs][android] Add Android Auto development instructions

### LINKED ISSUE #4719: Add document or links to install Android auto simulator on computer
Add document to install Android auto on computer

Tutorial is available here https://android-developers.googleblog.com/2015/08/announcing-android-auto-desktop-head.html

Just replace
`Enable Android Auto developer mode by starting the Android Auto companion app and tapping on the header image 10 times. This is a one-time step.`
By `Enable Android Auto developer mode by starting the Android Auto companion app and tapping on version 10 times. This is a one-time step.`

@AndrewShkrob congratulations for your work on Android Auto 🎉

### PR DESCRIPTION
**Pull request by rtsisyk**
Closes #4719

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -508,13 +531,13 @@ NDK version **26.1.10909125** and CMake version **3.22.1** are installed.
 If you are low on RAM, disk space or traffic there are ways to reduce system requirements:
 - exclude the `cpp` folder from indexing. If you do not make any work on the C++ code, this will greatly improve the start-up performance and the ram usage of Android Studio. Click on the `Project` tab on the left, find the `cpp` folder (should be next to the `java` folder), right click on it and select `Mark Directory as` -> `Excluded` (red folder icon). Then restart Android Studio.
 - in Android Studio enable "File > Power Save Mode";
+- disable thhe "Android NDK Support" plugin in "Settings -> Plugins" completely and use less the annoying Visual Studio Code or Qt Creator for editing C++ code instead;
```
Comment (pastk) [inline docs/INSTALL.md:534]: "thhe"
and please avoid subjective judging ("less annoying") in docs :)

Comment (pastk) [reply docs/INSTALL.md:534]: better use more neutral wording like
"* consider using less resource intensive IDEs to work with C++ code, e.g. Visual Studio Code or Qt Creator"

File: docs/INSTALL.md [hunk 2]
```diff
@@ -416,6 +416,29 @@ To enable logging in case of crashes, after installing a debug version, run:
 adb shell pm grant app.organicmaps.debug android.permission.READ_LOGS
 ```
 
+
+### Android Auto Development
+
+Android Auto can be developed and tested without having a physical device by using [Desktop Head Unit (DHU)](https://developer.android.com/training/cars/testing/dhu). Go to Android Studio > Tools -> SDK Manager -> SDK Tools to enable "Android Auto Desktop Head Unit". Desktop Head Unit version 2.0 build 2022-03-30-438482292 was tested.
+
+[Android Auto App](https://play.google.com/store/apps/details?id=com.google.android.projection.gearhead) is required for Auto functionality. The app should be installed from Google Play before connecting a phone to the Desktop Head Unit or a real car. Please don't waste time on trying to download and install that app manually on phones without Google Play unless you have genuine Google Play Services installed and root access to put the app into the system partition.
```
Comment (pastk) [inline docs/INSTALL.md:424]: "Please don't waste time on trying ..."
please keep it more technical like "AA doesn't work on phones without Google Play services"

File: docs/INSTALL.md (hunk 1 above)
Comment (rtsisyk) [reply docs/INSTALL.md:534]: I think that "crappy" is a more appropriate adjective in the context of IntelliJ/AndroidStudio.

Comment (rtsisyk) [reply docs/INSTALL.md]: Updated.

**Comment by pastk** [comment]
Needs a review from someone familiar with AA development.

**Comment by Jean-BaptisteC** [comment]
Maybe we can explain how to enable developper mode in Android Auto on mobile?

File: docs/INSTALL.md [hunk 3]
```diff
@@ -416,6 +416,29 @@ To enable logging in case of crashes, after installing a debug version, run:
 adb shell pm grant app.organicmaps.debug android.permission.READ_LOGS
 ```
 
+
+### Android Auto Development
+
+Android Auto can be developed and tested without having a physical device by using [Desktop Head Unit (DHU)](https://developer.android.com/training/cars/testing/dhu). Go to Android Studio > Tools -> SDK Manager -> SDK Tools to enable "Android Auto Desktop Head Unit". Desktop Head Unit version 2.0 build 2022-03-30-438482292 was tested.
```
Comment (AndrewShkrob) [inline docs/INSTALL.md:422]:
```suggestion
Android Auto can be developed and tested without having a physical device by using [Desktop Head Unit (DHU)](https://developer.android.com/training/cars/testing/dhu). Go to Android Studio > Tools -> SDK Manager -> SDK Tools and enable "Android Auto Desktop Head Unit".
```

It doesn't matter what version of DHU was used. It must work on all versions

**Review by AndrewShkrob — APPROVED** [review]

**Pull request merged**